SmartCharger.Data/Entities/User.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+
+namespace SmartCharger.Data.Entities
+{
+    public class User : BaseEntity
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string Email { get; set; }
+        public string Password { get; set; }
+        public bool? Active { get; set; }
+        public DateTime Created { get; set; }
```

Comment:
Same comment as with Charger class, I think this variable should be named differently.